server/src/core/sms/sms-provider.ts

```diff
@@ -0,0 +1,39 @@
+import createAtClient = require('africastalking');
+import { SmsService as AtSMSService, SendArgs } from 'africastalking-types';
```

Comment:
For consistency in naming `AtSMSService` should be `AtSmsService` (notice you also have `SmsService` instead of `SMSService`, or `At` instead of `AT`)